### PR TITLE
feat: add mapper

### PR DESCRIPTION
**Pull request by ronag**
I'm not too happy with the naming... suggestions?

Fixes: https://github.com/mcollina/hwp/issues/4

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -64,3 +64,4 @@ async function map (iterator, func, n = 16) {
 module.exports.forEach = forEach
 module.exports.mapIterator = mapIterator
 module.exports.map = map
+module.exports.mapper = (func, n = 16) => iterator => mapIterator(iterator, func, n)
```
Comment (mcollina) [inline index.js:67]: This needs a test!

**Review by mcollina — CHANGES_REQUESTED** [review]

**Comment by mcollina** [comment]
I'm ok with mapper.

@davidmarkclements is the master at naming things.

File: index.js (hunk 1 above)
Comment (ronag) [reply index.js:67]: Fixed

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**